pkg/k8s/apis/cilium.io/const.go

```diff
@@ -22,6 +22,9 @@ const (
 	// PolicyLabelName is the name of the policy label which refers to the
 	// k8s policy name
 	PolicyLabelName = "io.cilium.k8s-policy-name"
+	// PolicyLabelNamespace is the name of the namespace where the policy label
```

Comment:
I don't understand this comment. Can you clarify?

Reply:
"`PolicyLabelNamespace` is the namespace of the policy set in k8s", better?